android/java/org/chromium/chrome/browser/app/domain/CryptoModel.java

```diff
@@ -103,14 +97,6 @@ public void resetServices(Context context, TxService mTxService, KeyringService
             this.mAssetRatioService = mAssetRatioService;
             mPendingTxHelper.setTxService(mTxService);
             mNetworkModel.resetServices(mBraveWalletService, mJsonRpcService);
-            mPortfolioModel.resetServices(context, mTxService, mKeyringService, mBlockchainRegistry,
-                    mJsonRpcService, mEthTxManagerProxy, mSolanaTxManagerProxy, mBraveWalletService,
-                    mAssetRatioService);
-            if (mTransactionsModel != null) {
-                mTransactionsModel.resetServices(mContext, mTxService, mKeyringService,
-                        mBlockchainRegistry, mJsonRpcService, mEthTxManagerProxy,
-                        mSolanaTxManagerProxy, mBraveWalletService, mAssetRatioService);
-            }
         }
         init();
```

Comment:
These seem no longer needed:
```
mEthTxManagerProxy
mSolanaTxManagerProxy
mAssetRatioService
mBlockchainRegistry
```

Reply:
Also `SwapService` no longer needed in native android

Reply:
Because of the new Brave panel UI redesign these will be most probably re-used very soon. I prefer to leave them for now. Once the redesign is over I'll make sure to remove all the unneeded parts.